### ISSUE TITLE
⌨ Different navigation types produce different results for the same buttons

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13281]
### ⌨ Accessibility and Expected Behaviour

🛡️Fable testing results: 
Issue with labeling on the Sign in button. 

When using the Tab key, the shortcut F for form fields, or the shortcut B for buttons to navigate the form, my screen reader announces extra information for the "Sign In" button. I hear, "Sign in, connect to the GC Key Service." I cannot find this hint when using the up/down arrow keys. This causes some confusion for me.

### What operating systems are you encountering the problem on?

Windows

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

NonVisual Desktop Access (NVDA)

**Comment by petertgiles** [comment]
@BrookeCameron , sorry I don't understand the description and have no idea how to action this.  😅 

![Image](https://github.com/user-attachments/assets/431fcb2c-45ea-47f5-bd44-f1d65543dcca)

**Comment by BrookeCameron** [comment]
the _other_ sign in button on the sign in page 😅

**Comment by petertgiles** [comment]
> I cannot find this hint when using the up/down arrow keys.

Lol, OK, so I think I'm looking at the right button, now.  What is the trouble the user is having?  Is it confusing to have a button in the navbar that just says "sign in" and another button that goes to a different place that says "continue to GCKey and sign in"?

![Image](https://github.com/user-attachments/assets/446555d3-e6bd-44f0-81dc-b3633f8395e0)

**Comment by BrookeCameron** [comment]
from my understanding the user said that when using the arrows on the same button there is different copy. aka only hearing "sign in" when using the arrows instead of "Sign in, connect to the GC Key Service." when they are tabbing.

**Comment by petertgiles** [comment]
@esizer do you have any understanding of what is being described?

**Comment by esizer** [comment]
> [@esizer](https://github.com/esizer) do you have any understanding of what is being described?

heh, I know exactly what they are talking about. They are not referring to either of those because they are not buttons 😉 

They are referring to this button that uses the `title` attribute which is not well supported among screen readers so I'm not surprised it is giving them issues 🤷‍♀ 

![Image](https://github.com/user-attachments/assets/70ce9b27-bf5b-4774-ab41-9c855157a401)

**Comment by petertgiles** [comment]
Can't fix: GCKey problem  😿

**Comment by BrookeCameron** [comment]
Thanks for looking into this guys 🫶🏻
